### DISCUSSION TITLE
How to push modified Helm-Chart or open pull-request?

### DISCUSSION THREAD

**Discussion by jochen-schuettler** [sustainable-computing-io/kepler#1146] [Q&A]
I noticed kepler was still at v0.6.1 using the helm chart at kepler/kepler in repo https://sustainable-computing-io.github.io/kepler-helm-chart. I used helm repo pull kepler/kepler, untar, modified the kepler/Chart.yaml for release-0.7.2 (and updated version number to 0.5.4) and installed that with helm upgrade --namespace kepler --kube-context minikube kepler kepler. Happily this also fixed a bug in accessing eBPF. :smile:

Now I'd like to share this success with everybody, but I'm unsure how to push or open a pull-request for it. Likely I need to tar it with the updated version number as filename. And then ?

**Comment by jochen-schuettler** [comment]
Kepler 0.7.3 in the meantime, the helm-chart is still at 0.6.1.

**Reply by rootfs** [reply]
feel free to contribute to the repo here 
https://github.com/sustainable-computing-io/kepler-helm-chart

In the meantime, we are going to cut 0.7.4 this week.
